htdocs/css/styles_contrast.css

```diff
@@ -122,18 +126,23 @@ dl dt label {
 	font-size:15.3px;
 	background:#6e66b3;
 	margin:0;
-	padding:0.25em 1em;
-	border-top:1px solid #fff;
-	border-right:1px solid #aaa;
-	font-family: Tahoma;
+	padding:0.3em 1.5em;
+	border:2px solid #4e4780;
+	border-radius: 3px;
+	font-family: Trebuchet MS, Tahoma;
+	cursor: pointer;
+	transition-duration: 0.4s;
```

Comment:
:joy: fancy! It even [looks like this will work in older Firefox](https://caniuse.com/?search=transition-duration) if you add the `moz` version.